packages/blade/src/tokens/theme/bladeTheme.ts

```diff
@@ -11,2651 +11,843 @@ import {
-          },
-          link: {
-            default: globalColors.neutral.ashGrayLight[0],
-            hover: globalColors.neutral.ashGrayLight[300],
-            focus: globalColors.neutral.ashGrayLight[400],
-            active: globalColors.neutral.ashGrayLight[400],
-            disabled: globalColors.neutral.ashGrayLight.a500,
-            visited: globalColors.chromatic.orchid[400],
-          },
-        },
-        icon: {
-          primary: {
-            default: globalColors.chromatic.azure[600],
-            hover: globalColors.chromatic.azure[600],
-            focus: globalColors.chromatic.azure[600],
-            active: globalColors.chromatic.azure[600],
-            disabled: globalColors.neutral.ashGrayLight.a500,
-          },
-          secondary: {
-            default: globalColors.neutral.ashGrayLight[0],
-            hover: globalColors.neutral.ashGrayLight[0],
-            focus: globalColors.neutral.ashGrayLight[0],
-            active: globalColors.neutral.ashGrayLight[0],
-            disabled: globalColors.neutral.ashGrayLight.a500,
-          },
-          tertiary: {
-            default: globalColors.neutral.ashGrayLight[0],
-            hover: globalColors.neutral.ashGrayLight[0],
-            focus: globalColors.neutral.ashGrayLight[0],
-            active: globalColors.neutral.ashGrayLight[0],
-            disabled: globalColors.neutral.ashGrayLight.a500,
-          },
-          link: {
-            default: globalColors.neutral.ashGrayLight[0],
-            hover: globalColors.neutral.ashGrayLight[300],
-            focus: globalColors.neutral.ashGrayLight[400],
-            active: globalColors.neutral.ashGrayLight[400],
-            disabled: globalColors.neutral.ashGrayLight.a500,
-            visited: globalColors.chromatic.orchid[400],
-          },
-        },
+    overlay: {
+      background: {
+        moderate: globalColors.neutral.blueGrayLight.a200,
+        subtle: globalColors.neutral.black[200],
       },
     },
-    badge: {
+    popup: {
       background: {
-        blue: {
-          lowContrast: globalColors.chromatic.azure.a50,
-          highContrast: globalColors.chromatic.azure[600],
-        },
+        subtle: globalColors.neutral.blueGrayLight[0],
+        intense: globalColors.neutral.blueGrayLight[1000],
       },
       border: {
-        blue: {
-          lowContrast: globalColors.chromatic.azure.a200,
-          highContrast: globalColors.chromatic.azure[600],
-        },
-      },
-      text: {
-        blue: {
-          lowContrast: globalColors.chromatic.azure[600],
-          highContrast: globalColors.neutral.blueGrayLight[0],
-        },
-      },
-      icon: {
-        blue: {
-          lowContrast: globalColors.chromatic.azure[600],
-          highContrast: globalColors.neutral.blueGrayLight[0],
-        },
+        subtle: globalColors.neutral.blueGrayLight.a100,
+        Intense: globalColors.neutral.blueGrayLight[900],
```

Comment:
@kamleshchandnani this `Intense` is capitalised in few `border.Intense` places. Changing it from code for now but need to check plugin once